frontend/src/pages/Register.jsx

```
@@ -0,0 +1,105 @@
+        }
+    };
+
+    return (
+        <div className="flex justify-center items-center min-h-screen bg-gray-100">
+            <div className="card w-96 bg-white shadow-lg p-6 rounded-xl">
+                <h2 className="text-2xl font-bold text-center text-gray-700">Register</h2>
+                <form onSubmit={handleSubmit} className="mt-4">
+                    <div className="form-control">
+                        <label className="label">
+                            <span className="label-text">Username</span>
+                        </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)